app/views/comments/_comment.html.erb

```diff
@@ -13,3 +13,19 @@
   <%= link_to 'Edit', edit_article_comment_path(article_id: comment.article.id, id: comment.id) %>
 
 <hr />
+
+  <%= pluralize(comment.comments_raitings.count, 'person') %> scores this comment</br>
+    <% if comment.comments_raitings.sum(:raiting) > -5 %>
+      <%= comment.comments_raitings.sum(:raiting) %>
+    <% end %>
+
+  Like this comment:</br>
+
+    <% if current_user.rated_for?(comment).present?  %>
+      You've already rated this comment
+      <%= link_to "change your rate", article_comment_comments_rating_path(article_id: comment.article.id, comment_id: comment.id, id: current_user.rated_for?(comment), params: {raiting: 1}), method: :delete %>
```

Comment:
Zmieniłabym na "remove / delete / undo you rate" bo de facto chodzi o usunięcie, nie zmianę

Reply:
masz rację, zastanawiałam się, jak to nazwać